### PR TITLE
Import shropshire (closes #2390)

### LINKED ISSUE #2390: Import E06000051-Shropshire Council
Xpress DC

### PR DESCRIPTION
**Pull request by alexsdutton**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_shropshire.py [hunk 1]
```diff
@@ -20,16 +21,30 @@ def address_record_to_dict(self, record):
...
         if record.polling_place_id == "23961":
+            # Farlow & Oreton Village Hall, Fox Hill, Oreton, Kidderminster, Worcs
             record = record._replace(polling_place_easting="364411")
             record = record._replace(polling_place_northing="280033")
 
+        if record.polling_place_id == "23723":
+            # Nesscliffe Village Hall
+            record = record._replace(
+                polling_place_postcode="SY4 1AX", polling_place_uprn="10011838276"
+            )
```
Comment (GeoWill) [inline polling_stations/apps/data_collection/management/commands/import_shropshire.py]: I don't think any of these are the `polling_place_id`s from the current file. Sadly the councils' internal id's don't persist.

File: polling_stations/apps/data_collection/management/commands/import_shropshire.py [hunk 2]
```diff
@@ -20,16 +21,30 @@ def address_record_to_dict(self, record):
             rec["postcode"] = "TF10 9AP"
             rec["accept_suggestion"] = False
 
-        return rec
+        if uprn == "100071210835":
+            rec["postcode"] = "WV15 6NL"
+            rec["accept_suggestion"] = False
```
Comment (GeoWill) [inline polling_stations/apps/data_collection/management/commands/import_shropshire.py]: Should be a `return rec` here.

**Comment by GeoWill** [comment]
I just edited the PR's name to add the `(closes #2390)` - not sure if GH will do it's magic when it's not in the commit message :thinking: . Putting this here as a reminder to check when merging.

File: polling_stations/apps/data_collection/management/commands/import_shropshire.py (hunk 1 above)
Comment (alexsdutton) [reply polling_stations/apps/data_collection/management/commands/import_shropshire.py]: Urgh. I thought I updated those :/

**Comment by alexsdutton** [comment]
I think this was a terrible merge ust before commiting on my part, sorry.

**Pull request merged**